charts/signoz/templates/otel-collector/deployment.yaml

```diff
@@ -150,6 +153,9 @@ spec:
         - name: otel-collector-config-vol
           configMap:
             name: {{ include "otelCollector.fullname" . }}
+{{- if .Values.otelCollector.extraVolumes }}
+{{ toYaml .Values.otelCollector.extraVolumes | indent 8 }}
+{{- end }}
```

Comment:
same as above

Reply:
Same as your second comment

Reply:
Done in: https://github.com/SigNoz/charts/commit/25dfe02fbfa9bbce3ae37f6e9814224858b2dfcf and https://github.com/SigNoz/charts/commit/f131c9ed76a8f77f621640fe13e63edf9d2e909d